src/planscape/plan/models.py

```diff
@@ -70,6 +70,12 @@ class Project(models.Model):
     # Ratio of elevation to distance
     max_slope: models.FloatField = models.FloatField(null=True)
 
+    # Maximum cost per project in USD
+    max_cost_per_project_in_usd: models.FloatField = models.FloatField(null=True)
```

Comment:
Stray thought: will we need to update project area ranking logic to take these limits into consideration, too?

If so, maybe that can be added as an issue or TODO. 

Specifically, for ranking, the user provides project areas. In the forsys input dataframe, each project area represents a separate row. We should clarify how these constraints behave in that context.
option 1: these constraints are ignored for project area ranking.
option 2: project areas are filtered if their cost or acreage exceed these constraints.

Reply:
Agreed!  Added issue #758 for ranking by area and cost for future development.